core/cont/src/TClassTable.cxx

```diff
@@ -868,11 +957,12 @@ TNamed *ROOT::RegisterClassTemplate(const char *name, const char *file,
                                     Int_t line)
 {
    static TList table;
-   static Bool_t isInit = kFALSE;
-   if (!isInit) {
+   static Bool_t isInit = []() {
```

Comment:
Perhaps consider renaming `isInit`.

Reply:
I am not thinking of any better name.  The statement "`isInit == true` indicates that initialization is done" is still accurate.